src/qibolab/_core/instruments/qm/components/configs.py

```diff
@@ -33,17 +55,29 @@ class OpxOutputConfig(DcConfig):
 
     Possible values are -0.5V to 0.5V.
     """
-    filter: dict[str, list[float]] = Field(default_factory=dict)
-    """FIR and IIR filters to be applied for correcting signal distortions.
-
-    See
-    https://docs.quantum-machines.co/1.1.7/qm-qua-sdk/docs/Guides/output_filter/?h=filter#output-filter
-    for more details.
-    Changing the filters affects the calibration of single shot discrimination (threshold and angle).
-    """
     output_mode: Literal["direct", "amplified"] = "direct"
     sampling_rate: float = DEFAULT_SAMPLING_RATE
     upsampling_mode: Literal["mw", "pulse"] = "mw"
+    feedback_max: float = DEFAULT_FEEDBACK_MAX
+    feedforward_max: float = DEFAULT_FEEDFORWARD_MAX
+
+    @property
+    def filter(self):
+        feedback_filters = [
+            -i.feedback[1] for i in self.filters if isinstance(i, ExponentialFilter)
+        ]
+        return {
+            "filter": {
+                "feedback": normalize_feedback(feedback_filters, self.feedback_max)
+                if len(feedback_filters) > 0
+                else [],
+                "feedforward": normalize_feedforward(
+                    self.feedforward, self.feedforward_max
+                )
+                if len(self.feedforward) > 0
+                else [],
+            }
+        }
```

Comment:
I'm still missing the overall picture, but to me it is not intuitive that `OpxOutputConfig.filter` is a dictionary with a single key `"filter"` with some content. I'd rather expect the bare content, such that it could be used in places like
```py
{
  ...
  "filter": opx_output_config.filter
  ...
}
```

But I still need to read the rest of the PR.